mappings/net/minecraft/class_6497.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_6497
```

Comment:
Just curious, how did this get here? (does unmapping everything in Enigma do this?)

Reply:
Not sure, I saw that when I committed it too. I just used the "reset to obfuscated" thing on everything.